### PR TITLE
feat: enable k8sProcessor for RS infra mode

### PR DESCRIPTION
**Pull request by shekhar-rudder**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/processors/K8sProcessor.py [hunk 1]
```diff
@@ -212,9 +212,7 @@ def train(
             job_name=job_name, wh_creds=wh_creds, k8s_config=k8s_config, command=command
         )
         S3Utils.download_directory_from_S3(
-            s3_config["bucket"],
-            s3_config["region"],
-            s3_config["path"],
+            s3_config,
```
Comment (shekhar-rudder) [inline src/processors/K8sProcessor.py:215]: Signature of `download_directory_from_S3` was updated but all the callers were not updated

File: src/processors/K8sProcessor.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def _create_job(
                         containers=[
                             client.V1Container(
                                 name="container",
-                                image="rudderstack/profiles-classifier:v1",
+                                image="rudderstack/profiles-classifier:v0.1.0",
```
Comment (dpatchigolla) [inline src/processors/K8sProcessor.py:64]: @shekhar-rudder are you managing this docker image? Can you add some instructions in this repo readme on how to update this image ( Not urgent, you can add it as a backlog item )?

File: src/processors/K8sProcessor.py (hunk 1 above)
Comment (dpatchigolla) [reply src/processors/K8sProcessor.py:215]: So this is a bug fix? Are all the callers now fixed?

**Review by dpatchigolla — APPROVED** [review]

File: src/processors/K8sProcessor.py (hunk 1 above)
Comment (shekhar-rudder) [reply src/processors/K8sProcessor.py:215]: Pushed a fix for AWS as well

**Pull request merged**